### DISCUSSION TITLE
Post Css not working with basic html

### DISCUSSION THREAD

**Discussion by AyomiCoder** [tailwindlabs/tailwindcss#10976] [Help] [answered]
I can't use tailwind as a postcss without react or vue

**Comment by wongjn** [comment]
Sure you can! You could use [PostCSS CLI](https://www.npmjs.com/package/postcss-cli) to run PostCSS without a build system that React or Vue projects typically have. You could also use the [Tailwind CLI](https://tailwindcss.com/docs/installation) if you don't need PostCSS for anything else.

**Reply by wongjn** [reply]
> I'm having issues with the configuration and I have read the Upgrade docs. Still same[^1]

Consider providing what you have tried so far, so we can perhaps see where you may have some misconfiguration.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/10976#discussioncomment-5570036

**Reply by AyomiCoder** [reply]
I'm having issues with the configuration and I have read the Upgrade docs. Still same

**Reply by wongjn** [reply]
Consider providing what you have tried so far, so we can perhaps see where you may have some misconfiguration.

**Reply by AyomiCoder** [reply]
Share me a detailed guide you think I can follow

**Reply by wongjn** [reply, marked as answer]
For PostCSS CLI: https://www.npmjs.com/package/postcss-cli
For Tailwind CLI: https://tailwindcss.com/docs/installation

**Comment by AyomiCoder** [comment]


**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/10976#discussioncomment-5570155 to keep the reply chain together.
